src/server/frontend_wayland/xdg_output_v1.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2018 Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: William Wold <william.wold@canonical.com>
+ */
+
+#ifndef MIR_FRONTEND_XDG_OUTPUT_V1_H
+#define MIR_FRONTEND_XDG_OUTPUT_V1_H
+
+#include "xdg-output-unstable-v1_wrapper.h"
```

Comment:
A lot of what is in this file could safely be moved to the .cpp. All that is needed here is a factory function similar to `create_wl_shell()` or `create_data_device_manager()`.

Reply:
Done